lib/luatest_server.py

```diff
@@ -85,6 +88,21 @@ def deploy(self, vardir=None, silent=True, wait=True):
         self.vardir = vardir
         if not os.access(self.vardir, os.F_OK):
             os.makedirs(self.vardir)
+        if self.lua_libs:
+            for i in self.lua_libs:
+                source = os.path.join(self.testdir, i)
+                try:
+                    if os.path.isdir(source):
+                        shutil.copytree(source,
+                                        os.path.join(self.vardir,
+                                                     os.path.basename(source)))
+                    else:
+                        shutil.copy(source, self.vardir)
+                except IOError as e:
+                    if (e.errno == errno.ENOENT):
+                        continue
+                    raise
+            os.environ["LUA_PATH"] = self.vardir + '/?.lua;' + os.environ.get("LUA_PATH", ";;")
```

Comment:
If `LUA_PATH` is not defined, you will get `;;;` at the end of the path. Looks like it should be `os.environ.get("LUA_PATH", ";")`.

Reply:
fixed